python/tvm/relay/frontend/tflite.py

```diff
@@ -1124,10 +1124,14 @@ def convert_conv(self, op, conv_type):
             pad_left, pad_right = get_pad_value(input_w, dilated_kernel_w, stride_w)
             do_pad = not (pad_top == 0 and pad_bottom == 0 and pad_left == 0 and pad_right == 0)
             if do_pad:
+                pad_value = 0
+                if input_tensor.qnn_params:
+                    pad_value = get_scalar_from_constant(input_tensor.qnn_params['zero_point'])
                 in_expr = _op.nn.pad(data=in_expr, pad_width=((0, 0),
                                                               (pad_top, pad_bottom),
                                                               (pad_left, pad_right),
-                                                              (0, 0)))
+                                                              (0, 0)), pad_value=float(pad_value))
```

Comment:
Must this be float(pad_value)? For QNN, this is int type. Does our pad require it be float type?

Reply:
Our convert_pad is not correct too for QNN. We should use OutputZeroPoint for pad_value, please fix it together and write one unit testing for it.

Reply:
So before giving the final output, in_expr goes first through the padding and then the result of the _op.pad is passed to _op.conv2d. My concern here is that if we set  pad_value=output_zero_point then wouldn't that be a problem later in _op.conv2d where we do the operation with the input_tensor qnn parameters. Correct me if I am wrong.

Reply:
Zhao meant just single pad operator, not the pad in conv. I will fix it.

Reply:
> Must this be float(pad_value)? For QNN, this is int type. Does our pad require it be float type?

Pad only takes float value for now. Internaly it gets converted to the input dtype. So, this is fine for now.